irr/src/OpenGL/Driver.cpp

```diff
@@ -261,16 +261,6 @@ bool COpenGL3DriverBase::genericDriverInit(const core::dimension2d<u32> &screenS
 
 	StencilBuffer = stencilBuffer;
 
-	DriverAttributes->setAttribute("MaxTextures", (s32)Feature.MaxTextureUnits);
-	DriverAttributes->setAttribute("MaxSupportedTextures", (s32)Feature.MaxTextureUnits);
-	DriverAttributes->setAttribute("MaxAnisotropy", MaxAnisotropy);
-	DriverAttributes->setAttribute("MaxIndices", (s32)MaxIndices);
-	DriverAttributes->setAttribute("MaxTextureSize", (s32)MaxTextureSize);
-	DriverAttributes->setAttribute("MaxArrayTextureLayers", (s32)MaxArrayTextureLayers);
```

Comment:
FWIW at least this one cannot be retrieved in any other way.
but we can of course just add a struct getter when needed.

Reply:
> but we can of course just add a struct-like getter when needed.

yep, that was my thought. initially i wanted to replace this with such a thing, but i figured it would currently be unused, and thus kinda arbitrary.